src/cascadia/TerminalControl/TermControl.h

```diff
@@ -94,6 +96,10 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         //      viewport via touch input.
         std::optional<winrt::Windows::Foundation::Point> _touchAnchor;
 
+        TIMESTAMP _lastMouseClick;
+        bool _doubleClickOccurred;

```

Comment:
These can be initialized here with default values.  Currently, these are not initialized in the constructor.

Reply:
@carlos-zamora Curious why we are doing these in the initializer list instead of the header.

Reply:
@carlos-zamora Looking at this closer - Technically this isn't "new" code and the class already existed so having them in the initializer list is OK since it matches the "old" style.   Sorry for the confusion.

Reply:
No worries. Good to know :)